src/transforms/mapperTransforms.ts

```diff
@@ -716,6 +730,7 @@ export function getMapperTypeFromSchema(type: SchemaType, format?: string) {
     case SchemaType.Number:
       return MapperType.Number;
     case SchemaType.Object:
+    case SchemaType.Any:
```

Comment:
Hmm, should this be the "any" mapper type instead of "object"? They get serialized differently:
https://github.com/Azure/azure-sdk-for-js/blob/1e02861f3abec278cd5b3629b5737760727878ad/sdk/core/core-http/src/serializer.ts#L127-L130

That would mean adding "any" to the `MapperType` enum in core-http:
https://github.com/Azure/azure-sdk-for-js/blob/1e02861f3abec278cd5b3629b5737760727878ad/sdk/core/core-http/src/serializer.ts#L1036
but we could test the impact of this by returning the string "any" in the meantime.

Reply:
Good catch, try that out. 

It like core-http is exporting MapperType twice: https://github.com/Azure/azure-sdk-for-js/blob/1e02861f3abec278cd5b3629b5737760727878ad/sdk/core/core-http/src/serializer.ts#L901

Reply:
Thanks for the idea Chris, I have moved to use "any" string instead of Object

Reply:
Yeah, not sure why that's the case, but they have very different shapes. I got really confused when I first saw that.